pkdiagram/views/addanythingdialog.py

```
@@ -357,283 +324,286 @@ def _addEvent(self):
         """
         Only here to be easily wrapped in a macro.
         """
-        _log.debug(f"AddAnythingDialog.onDone: {self.item.property('kind')}")
 
-        if self.item.property("kind") is None:
-            kind = None
-        else:
-            kind = EventKind(self.item.property("kind"))
+        _log.debug(f"AddAnythingDialog._addEvent()")
```

Ruff (0.13.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)